### PR TITLE
Improve operation example

### PR DESCRIPTION
**Pull request by aganglada**
Based on the conversation we had in [this issue](https://github.com/mswjs/msw/issues/428) I want to improve the `graphql.operation` method example for future reference.

### CODE REVIEW

File: docs/api/graphql/operation.mdx [hunk 1]
```diff
@@ -55,12 +55,16 @@ const root = {
 graphql.operation(async (req, res, ctx) => {
   const payload = await graphqlRequest(
     schema,
-    req.body,
+    req.body.query,
     root,
     null,
-    req.variables,
+    req.body.variables,
```
Comment (kettanaito) [inline docs/api/graphql/operation.mdx:61]: GraphQL operation variables are parsed and always present in `req.variables`:

https://github.com/mswjs/msw/blob/d48a2afb892e7abfefb7f099097d644a1ca6dbc9/src/graphql.ts#L165-L170

Referencing them in a `req.body` is okay for POST requests, but if you're communicating with a GraphQL server using a GET request, then `req.body` will be `undefined` and this expression throws. `req.variables` has an isomorphic behavior in this sense.

File: docs/api/graphql/operation.mdx [hunk 2]
```diff
@@ -55,12 +55,16 @@ const root = {
...
   )
 
-  return res(payload)
+  if (payload.errors) {
+    return res(ctx.errors(payload.errors))
+  }
+
+  return res(ctx.data(payload))
```
Comment (kettanaito) [inline docs/api/graphql/operation.mdx:68]: If `payload` represents the entire operation payload, then we should pass `payload.data` to the `ctx.data` utility.

File: docs/api/graphql/operation.mdx [hunk 3]
```diff
@@ -55,12 +55,16 @@ const root = {
...
-    req.body,
+    req.body.query,
     root,
     null,
-    req.variables,
+    req.body.variables,
   )
 
-  return res(payload)
+  if (payload.errors) {
```
Comment (kettanaito) [inline docs/api/graphql/operation.mdx:64]: You don't necessarily have to check for errors, as in GraphQL `errors` will be undefined if there are none. Containing `errors: undefined` in the response is okay, as far as I know. 

That's to the point that we can illustrate something like this for the sake of simplicity:

```js
const { data, errors } = payload

return res(
  ctx.data(data),
  ctx.errors(errors),
)
```

**Review by kettanaito — COMMENTED** [review]
Hey, @aganglada! Thank you for submitting these improvements. I've left a few comments and a suggestion, could you please let me know what you think of them?

File: docs/api/graphql/operation.mdx (hunk 3 above)
Comment (aganglada) [reply docs/api/graphql/operation.mdx:64]: Screenshot 2020-10-20 at 09 10 50
Typescript doesn't really like this on `ctx.errors`, should I send a PR fixing it?

Comment (aganglada) [reply docs/api/graphql/operation.mdx]: https://github.com/mswjs/msw/pull/430
Already did it, I think it make sense 😃

**Comment by aganglada** [comment]
Hey @kettanaito I guess this will be merged, when the new version is released right? Not to confuse people 😃

**Review by kettanaito — APPROVED** [review]
Thank you for this addition, @aganglada!

**Pull request merged**